python/lsst/ctrl/bps/wms/panda/panda_service.py

```diff
@@ -196,8 +201,7 @@ def from_generic_workflow(cls, config, generic_workflow, out_prefix, service_cla
         idds_workflow.generated_tasks = workflow_generator.define_tasks()
         cloud_prefix = config['bucket'] + '/' + \
             config['payload_folder'] + '/' + config['workflowName'] + '/'
-        for task in idds_workflow.generated_tasks:
-            cls.copy_pickles_into_cloud(task.local_pfns, cloud_prefix)
+        cls.copy_pickles_into_cloud([config['bps_defined']['run_qgraph_file']], cloud_prefix)
```

Comment:
This is very fragile code as opposed to checking the inputs for the jobs.  I expect that this section will be revisited in the upcoming execution butler work.  We should talk then to figure out if there are GenericWorkflow issues that need to be fixed so that this can be written more generically.